server/controllers/__tests__/courseCtrl.test.js

```diff
@@ -58,6 +58,8 @@ jest.mock('../../apiCalls/koppsCourseData', () => ({ getKoppsCourseData: () => (
 jest.mock('../../apiCalls/ugRestApi', () => ({
   getCourseEmployees: jest.fn(() => ({ examiners: '<p>Examiner 1 </p>' })),
 }))
+// `errorResponse` is clearly used below
```

Comment:
Is it clearly used? It is assigned, but never read (after you removed ` if (errorResponse) console.error('errorResponse', errorResponse)` below).

Reply:
Fixed c77f3af